cmd/apps/sealed_secret_app.go

```diff
@@ -90,11 +73,26 @@ func MakeInstallSealedSecrets() *cobra.Command {
 			return err
 		}
 
-		err = helm.Helm3Upgrade("stable/sealed-secrets",
-			namespace, "values.yaml", defaultVersion, overrides, wait)
+		sealedSecretAppOptions := types.DefaultInstallOptions().
+			WithNamespace(namespace).
+			WithHelmPath(path.Join(userPath, ".helm")).
+			WithHelmRepo("stable/sealed-secrets").
+			WithHelmURL("https://charts.helm.sh/stable").
```

Comment:
Bitnami should have a new home for the chart. Can you look around and see if you can find it? The "stable helm repo" is going to be deprecated shortly.